File: JAVA Programs/Program251.java
import java.util.*;

class Program251
{
    public static int countCapital(String str)
    {
        char Arr[]=str.toCharArray();
        int iCount=0;

        for(int i=0; i<Arr.length;i++)
        {
            if((Arr[i]>='A')&&(Arr[i]<='Z'))
            {
                iCount++;
            }
        }
        return iCount;
    }

    public static void main(String args[])
    {
        Scanner sobj=new Scanner(System.in);

        System.out.println("Enter your name: ");
        String name=sobj.nextLine();

        int iRet=0;

        iRet=countCapital(name);

        System.out.println("Number of capital characters are: "+iRet);

    }
}

File: Assignment25/Program25_1.c
// Write a program which accept string from user and count number of capital characters.

#include<stdio.h>

///////////////////////////////////////////////
//
// Function Name: CountCapital
// Description: Return number of capital characters
// Input: Character
// Output: Integer
// Author: Nikita Sagar Rane
// Date: 31/05/2024
//
///////////////////////////////////////////////

int CountCapital(char *str)
{
    int iCount=0;

    while(*str != '\0')
    {
        if(( *str>='A')&&( *str <='Z'))
        {
            iCount++;
        }
        str++;
    }
    return iCount;
}

int main()
{
    char arr[20];
    int iRet=0;

    printf("Enter string: ");
    scanf("%[^'\n']s",arr);

    iRet=CountCapital(arr);
    printf("Number of capital letters are: %d",iRet);

    return 0;
}